pywbemtools/pywbemcli/_pywbem_server.py

```diff
@@ -551,15 +549,13 @@ def connect(self, log=None, use_pull=None, verbose=None):
           ValueError: Other issues with server-related attributes of this
             object that are considered programming errors.
         """
-        # print("Debug: PywbemServer connect")
+        ctx = click.get_current_context()
+        if ctx.obj:
+            if ctx.obj.verbose:
+                click.echo("Connect to server {}".
+                           format(self._mock_server or self._server))
```

Comment:
The term "server" is ambiguous here. I suggest to be specific about mock environment vs. WBEM server, e.g. by changing the lines with the click.echo() call to:
```
            if self._mock_server:
                server_txt = "mock environment {}".format(self._mock_server)
            else:
                server_txt = "WBEM server {}".format(self._server)
            click.echo("Connecting to {}".format(server_txt))
```